firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1218,6 +1218,11 @@ extension String {
                 tableName: "Onboarding",
                 value: "3. Select *%@*",
                 comment: "The third label on the Default Browser Popup, which is a card with instructions telling the user how to set Firefox as their default browser. Placeholder is the app name. The *text inside asterisks* denotes part of the string to bold, please leave the text inside the '*' so that it is bolded correctly.")
+            public static let DescriptionFooter = MZLocalizedString(
+                key: "DefaultBrowserPopup.DescriptionFooter.v124",
+                tableName: "Onboarding",
+                value: "*Is %@ already your default?* Close this message and tap Skip.",
```

Comment:
We support bold through *bold* ?

Reply:
we do, look above the file

Reply:
Yes I am asking, first time I see this format in a string 👀

Reply:
I guess it's only supported for Nimbus onboarding strings?

Reply:
Yeah, I saw it at couple of places. 

FYI @adudenamedruby feel free to update if this isn't the place for it as I saw you added others like this as well